contracts/RMRK/equippable/RMRKEquippable.sol

```diff
@@ -373,16 +375,17 @@ contract RMRKEquippable is RMRKNesting, IRMRKEquippable {
         internal
         virtual
     {
-        _tokenApprovalsForResources[tokenId] = to;
-        emit ApprovalForResources(ownerOf(tokenId), to, tokenId);
+        address owner = ownerOf(tokenId);
+        _tokenApprovalsForResources[tokenId][owner] = to;
+        emit ApprovalForResources(owner, to, tokenId);
```

Comment:
Am I correct to assume that access control has to be handled by the implementer?

Reply:
For most operations we already provide access control to the token owner or approved only.